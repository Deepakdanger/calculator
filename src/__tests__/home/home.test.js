import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../components/home';

it('render home UI correctly', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
