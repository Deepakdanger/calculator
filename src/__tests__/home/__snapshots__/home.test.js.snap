// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home UI correctly 1`] = `
<div
  className="homes"
>
  <nav
    className="nav1"
  >
    <p
      className="logo11"
    >
      Math Magic
    </p>
    <ul
      className="right"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
  <h1>
    Welcome to our Page!
  </h1>
  <p
    className="para"
  >
    This is a math magician website which provide calculator as well as inspiring quote.
  </p>
</div>
`;
